src/organizations/components/OrgProfileTab/SuspendPaidOrgOverlay.tsx

```diff
@@ -158,6 +160,16 @@ export const SuspendPaidOrgOverlay: FC = () => {
         >
           You will be able to recover this organization's data for up to 7 days
           by contacting support. It will be unrecoverable afterwards.
+          {params.users > 0 && (
+            <>
+              <br />
+              <br />
+              <Link to={`/orgs/${org.id}/members`} onClick={onClickCancel}>
+                {params.users} users in this organization
+              </Link>{' '}
+              will lose access to all organization data immediately.
+            </>
+          )}
```

Comment:
In the video, we see `0 users will lose access to all organization data immediately`. How is that happening?  Isn't the warning only shown when `params.users > 0`?

Reply:
yup you're correct. Ignore the video for now. I'll have to add a new video that reflects the most recent commit.